lib/matplotlib/axes/_axes.py

```diff
@@ -1975,6 +1975,8 @@ def bar(self, left, height, width=0.8, bottom=None, **kwargs):
         if not self._hold:
             self.cla()
         color = kwargs.pop('color', None)
+        if color is None:
```

Comment:
We are going to need to do the keyword normalization prior to this step. How do we want to coordinate this with the normalization PR?